src/app/cluster/list/external-cluster/template.html

```diff
@@ -100,7 +100,7 @@
       <ng-container matColumnDef="created">
         <th mat-header-cell
             *matHeaderCellDef
-            class="km-header-cell p-20">Created
```

Comment:
@imharshita this update will be reflected for both even if user tries to create via wizard.